### ISSUE TITLE
Move App Data to SQL (Postgres)

### ISSUE THREAD

**Issue by benedeki** [AbsaOSS/enceladus#1201]
## Background
Having the application data stored in Postgress SQL DB would bring considerable benefits

**Comment by Zejnilovic** [comment]
Could we close #19  then?

**Comment by benedeki** [comment]
> 
> 
> Could we close #19 then?

What about making the issue DB tech independent? I think the integration tests would still come handy.

**Comment by GeorgiChochov** [comment]
> > Could we close #19 then?
> 
> What about making the issue DB tech independent? I think the integration tests would still come handy.

Yes, the integration tests should be independent from the actual database of choice.
